sdk/examples/xo_python/tests/test_tp_xo_rust.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2017 Intel Corporation
```

Comment:
Is this intentionally 2017 intel or is that copy/paste error?

Reply:
Definitely a copy/paste error... Will fix this to 2018 Bitwise IO, Inc.